src/dotnet/HQ.Server/Services/ProjectStatusReportServiceV1.cs

```diff
@@ -384,6 +395,7 @@ public async Task<Result<GetProjectStatusReportTimeV1.Response>> GetProjectStatu
         {
             Records = await sorted.ToListAsync(ct),
             Staff = await staff.ToListAsync(ct),
+            ProjectId = project != null ? project.Id : Guid.Empty,
```

Comment:
Avoid using `Guid.Empty`, if we dont have a value we should use `null`.  Guid.Empty can lead to hidden issues where at compile time everything looks right but no record should exist in the DB with an ID of `0000-...-0000`.